src/pages/Market/components/ItemCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import likeicon from '../../../assets/images/likeicon.png';
+import '../Market.css';
+function ItemCard( { item }) {
+    return (
+        <div className="card-container">
+            <img className="item-img" src={item.images[0]} alt={item.name}></img>
+            <div className="item-info">
+                <div className="title">{item.name}</div>
+                <div className="price">{item.price}원</div>
```

Comment:
P2:
디자인상에서, 그리고 일반적으로 돈을 표시할때 정해진 자리마다 콤마를 찍어줍니다.
자바스크립트 내장 메소드를 사용하면 편리하게 변환할 수 있습니다.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

```suggestion
           <div className="price">{item.price.toLocaleString()} 원</div>
```